### PR TITLE
[WIP] Add ci and conda recipe

### PR DESCRIPTION
**Pull request by raydouglass**

### CODE REVIEW

**Comment by raydouglass** [comment]
rerun tests

File: ci/cpu/build.sh [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+}
+
+# Set path and build parallel level
+export PATH=/conda/bin:/usr/local/cuda/bin:$PATH
+export PARALLEL_LEVEL=4
+
+# Set versions of packages needed to be grabbed
+export CUDF_VERSION=0.8.*
+export NVSTRINGS_VERSION=0.8.*
+export RMM_VERSION=0.8.*
```
Comment (dillon-cullinan) [inline ci/cpu/build.sh]: These should all be deleted.

File: ci/cpu/prebuild.sh [hunk 2]
```diff
@@ -0,0 +1 @@
+#!/usr/bin/env bash
```
Comment (dillon-cullinan) [inline ci/cpu/prebuild.sh]: Does this package need to be uploaded for all CUDA/Python combos?

Comment (raydouglass) [reply ci/cpu/prebuild.sh]: It's python only, so yes.

**Pull request merged**